CONTRIBUTING.rst

```diff
@@ -274,6 +274,94 @@ Schedule some tasks with:
 
     (venv) $ python -m procrastinate_demo
 
+Use Docker for Procrastinate development
+----------------------------------------
+
+In the development setup described above, Procrastinate, its dependencies, and the
+development tools (``tox``, ``black``, ``pytest``, etc.) are installed in a virtual
+Python environment on the host system. Alternatively, they can be installed in a Docker
+image, and Procrastinate and all the development tools can be run in Docker containers.
+Docker is useful when you can't, or don't want to, install System requirements such as
+the ``libpq-dev`` package (required by the ``psycopg2`` dependency).
+
+This section shows, through ``docker-compose`` command examples, how to test and run
+Procrastinate in Docker.
+
+Build the ``procrastinate`` Docker image:
+
+.. code-block:: console
+
+    $ docker-compose build procrastinate
+
+Run the automated tests:
+
+.. code-block:: console
+
+    $ export UID
+    $ export GID=$(id -g)
+    $ docker-compose run --rm procrastinate pytest
+
+The ``UID`` and ``GID`` environment variables are set and exported for the Procrastinate
+container to be run with the current user id and group id. If not set or exported, the
+Procrastinate container will run as root, which isn't recommended.
```

Comment:
(I think this statement should be sourced. It's not universally recognized as true)

Reply:
Actually, there's a real problem related to running the container as root: that will create in the developer's repo files and directories owned by root, which is certainly not desired. I'll change the wording.

Reply:
At what point does procrastinate creates a file ?

Reply:
`mypy` creates a `.mypy_cache` dir for example.